### ISSUE TITLE
Add doctests target to tox

### ISSUE THREAD

**Issue by grzanka** [DataMedSci/beprof#63]

**Comment by antnieszka** [comment]
Which version of python? Or should we run them with normal tests on every python version?

**Comment by grzanka** [comment]
I'd recommend to test on: 2.7, 3.2 - 3.7.

**Comment by antnieszka** [comment]
@grzanka I'm trying to run this in a bit dirty way: https://travis-ci.org/ant6/beprof/builds/182900341
```
[testenv:doctests]
deps = -rrequirements.txt
commands = python -m doctest -v beprof/curve.py beprof/profile.py beprof/functions.py
```

**Comment by antnieszka** [comment]
Well, py3.2 fails...
`Requests officially supports Python 2.6–2.7 & 3.3–3.5, and runs great on PyPy.`

**Comment by grzanka** [comment]
`VENVVER="<14.0.0" PIPVER="<8.0.0" STVER="<30.0.0"` should help you

**Comment by antnieszka** [comment]
It did 😄 All doctests have passed - https://travis-ci.org/ant6/beprof/builds/182904473

**Comment by antnieszka** [comment]
Closed in #121
